interop-testing/src/test/java/io/grpc/testing/integration/RetryTest.java

```diff
@@ -103,8 +105,11 @@ public class RetryTest {
   @Rule
   public final GrpcCleanupRule cleanupRule = new GrpcCleanupRule();
   private final FakeClock fakeClock = new FakeClock();
-  @Mock
-  private ClientCall.Listener<Integer> mockCallListener;
+  private TestListener realMockCallListener = new TestListener();
```

Comment:
I wouldn't use the word "mock" since it's not a mock object. Just `testCallListener` would do I think, or people ofthen use the term "fake" with these kinds of classes.

Reply:
changed to testCallListener